jwst/source_catalog/source_catalog.py

```diff
@@ -966,7 +964,20 @@ def catalog(self):
         catalog = self.format_columns(catalog)
 
         # update metadata
-        self.meta['version']['jwst'] = jwst_version
```

Comment:
I liked your suggestion in: https://github.com/spacetelescope/jwst/pull/8014#issuecomment-1799534714 to remove this metadata modification. Is there any objection to just deleting this line (and not replacing it with the code here)?

Deleting this will mean that the `jwst` version will not be in the `meta.version(s)` metadata of the `ecsv` containing the source catalog. As the [source catalog documentation](https://jwst-pipeline.readthedocs.io/en/latest/jwst/source_catalog/main.html#source-catalog-table) doesn't describe this metadata and I don't see anything in the pipeline that uses it the change in output seems minor. This frees up photutils to change the metadata structure (which is copied to the ecsv outout in line 238).